packages/block-editor/src/components/block-settings-menu-controls/index.js

```diff
@@ -40,10 +35,9 @@ const BlockSettingsMenuControlsSlot = ( {
 			const ids =
 				clientIds !== null ? clientIds : getSelectedBlockClientIds();
 			return {
-				selectedBlocks: map(
-					getBlocksByClientId( ids ).filter( Boolean ),
-					( block ) => block.name
-				),
+				selectedBlocks: getBlocksByClientId( ids )
+					.filter( Boolean )
```

Comment:
Always an array since we're calling it on a result of `Array.prototype.filter()`.